src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch"
+
+export function fetchCollectorProfile(accessToken: string): any {
```

Comment:
this returns a `Body` from the fetch lib, so should be getting typed properly.

I wasn't able to figure out how to import/type that properly tho so went with `any`.